app/scripts/lib/xhr.js

```diff
@@ -67,6 +67,7 @@ define([
         // make sure to set the dataType for Firefox <21. See issue #1930
         dataType: 'json',
         type: options.type,
+        timeout: options.timeout || 0,
```

Comment:
> its the JQuery default when a timeout isnt set, so I thought we can use that here as well.

Let's just pass `options.timeout` without the `|| 0`, otherwise we could inadvertently modify the default behavior of jQuery or whatever low level XHR library we decide to use.